src/languages/swedish.rs

```diff
@@ -0,0 +1,40 @@
+use super::super::{Language, TimeUnit};
+
+/// Default language for timeago
```

Comment:
It's not a default language for timeago.

Reply:
oops

Reply:
Already removed myself anyway.